Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41355&attempt_id=241a1876-599f-4cdc-862e-493e08e1b5f5)

Attempted: pythonPackages.pykdtree, python3Packages.pykdtree

<details><summary>Partial log (click to expand)</summary><p>

```
pykdtree/kdtree.c:529:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
1 warning and 1 error generated.
1 warning and 1 error generated.
error: command 'clang' failed with exit status 1
error: command 'clang' failed with exit status 1
builder for '/nix/store/q8ddc3mb723lhfhcrymk2m3pxascziif-python2.7-pykdtree-1.3.0.drv' failed with exit code 1
builder for '/nix/store/hyifzfa4q7jrb2881mgsab4ad34592q4-python3.6-pykdtree-1.3.0.drv' failed with exit code 1
error: build of '/nix/store/hyifzfa4q7jrb2881mgsab4ad34592q4-python3.6-pykdtree-1.3.0.drv', '/nix/store/q8ddc3mb723lhfhcrymk2m3pxascziif-python2.7-pykdtree-1.3.0.drv' failed
```
</p></details>

--- a/pkgs/development/python-modules/pykdtree/default.nix
+++ b/pkgs/development/python-modules/pykdtree/default.nix
@@ -9,6 +9,8 @@ buildPythonPackage rec {
     sha256 = "79351b79087f473f83fb27a5cd552bd1056f2dfa7acec5d4a68f35a7cbea6776";
   };
 
+  preInstall = ''export USE_OMP=0'';
+
   propagatedBuildInputs = [ numpy ];
 
   checkInputs = [ nose ];

--- a/pkgs/development/python-modules/pykdtree/default.nix
+++ b/pkgs/development/python-modules/pykdtree/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, buildPythonPackage, fetchPypi, numpy, nose }:
+{ stdenv, buildPythonPackage, fetchPypi, numpy, nose, openmp }:
 
 buildPythonPackage rec {
   pname = "pykdtree";
@@ -9,7 +9,7 @@ buildPythonPackage rec {
     sha256 = "79351b79087f473f83fb27a5cd552bd1056f2dfa7acec5d4a68f35a7cbea6776";
   };
 
-  preInstall = ''export USE_OMP=0'';
+  buildInputs = [ openmp ];
 
   propagatedBuildInputs = [ numpy ];
 
@@ -22,4 +22,3 @@ buildPythonPackage rec {
     maintainers = with maintainers; [ psyanticy ];
   };
 }
-

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -382,7 +382,9 @@ in {
     callPackage = pkgs.callPackage;
   };
 
-  pykdtree = callPackage ../development/python-modules/pykdtree { };
+  pykdtree = callPackage ../development/python-modules/pykdtree {
+    inherit (pkgs.llvmPackages) openmp;
+  };
 
   pyparser = callPackage ../development/python-modules/pyparser { };
 

